deb/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+# You must remove unused comment lines for the released package.
```

Comment:
`debmake` usually creates these Debian helper scripts and I copied the template to be conform (and safe). But there has nothing to be done here, because we are not publishing the deb file in an official Debian repo.